libs/@local/repo-chores/scripts/check-license-in-workspaces.ts

```diff
@@ -49,7 +49,7 @@ const script = async () => {
 
   for (const licenseFilePath of licenseFilePaths) {
     const licenseFileName = path.basename(licenseFilePath);
-    if (!licenseFileName.match(/^LICENSE(-[A-Z0-9]+)?.md$/)) {
+    if (!licenseFileName.match(/^LICENSE(-[A-Z0-9]+)?(\.(md|txt))?$/)) {
```

Comment:
Converts

```
🔴 [NAMING]  apps/engine/lib/memory/src/arrow/util/LICENSE
🔴 [NAMING]  apps/engine/lib/memory/src/arrow/ipc/serialize/LICENSE
```

```
🟡 [EXTRA]   apps/engine/lib/memory/src/arrow/util/LICENSE
🟡 [EXTRA]   apps/engine/lib/memory/src/arrow/ipc/serialize/LICENSE
```

These two files do not use markdown are external, which seems reasonable to tolerate. LICENSE files within worksppaces still need to have `md` extension.

Reply:
I feel like this should conditionally read 'EXTRA' only when the files exist outside of an expected workspace location, and if they occur within one they should be red 'NAMING' errors.

These are fine to mark as extra, though, as they are the first case of the two.

Reply:
Possibly, I just think that any further improvements have diminishing returns so are not worth extra time, at least in the near future. If you disagree, happy to pick a new ticket!

Reply:
If not simple enough to ship as part of this (which has now already merged, so nm) we can ticketize for a future cleanup week. See [ticket](https://app.asana.com/0/1203193968491715/1203767287750344/f) (_internal link_).